app/views/ev_map/show.html.erb

```diff
@@ -7,12 +7,26 @@
     attribution: 'Map data &copy; <a href="https://www.openstreetmap.org/">OpenStreetMap</a> contributors, <a href="https://creativecommons.org/licenses/by-sa/2.0/">CC-BY-SA</a>, Imagery © <a href="https://www.mapbox.com/">Mapbox</a>',
     maxZoom: 20,
     id: 'mapbox.streets',
-    accessToken: 'pk.eyJ1IjoibWFyeWFncmFjZSIsImEiOiJjanM0eG9qMmIwOXlhNDVuaTNuOXMzcXd1In0.Y3D7BRb8btkvg5JV_6PZaQ'
+    accessToken: '<%= ENV['MAPBOX_KEY'] %>'
```

Comment:
Glad that we got the ENV variable working in the script!

Reply:
Yeh perfect thanks guys for getting that outta there